components/src/dynamo/vllm/multimodal_utils/threaded_micro_batcher.py

```
@@ -0,0 +1,468 @@
+            stopping = self._state is not _State.RUNNING
+        if stopping:
+            for work in runnable:
+                self._consume(
+                    work, error=RuntimeError("ThreadedMicroBatcher shut down")
+                )
+            return
+        items = [w.item for w in runnable]
+        try:
+            results = self._fn(items)
+        except (
+            BaseException
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `BaseException`

(BLE001)